Fix In Batch File

vinothbabu13 opened the pull request:
`Getting PHP Warning: Declaration of Cassandra\Request\Batch::queryParameters($consistency, array $options = Array) should be compatible with Cassandra\Request\Request::queryParameters($consistency, array $values = Array, array $options = Array) in /var/www/html/php-cassandra/src/Request/Batch.php on line 122 for PHP7`

vinothbabu13 committed:
error fix

pincher2012 commented on src/Response/Exception.php:
```diff
@@ -1,4 +1,4 @@
 
-class Exception extends \Cassandra\Exception {}
\ No newline at end of file
+class Exception implements \Cassandra\Exception {}
```
Why implements? \Cassandra\Exception is class but not interface.

vinothbabu13 committed:
fix for the error - FatalErrorException in Exception.php line 4:
Class Cassandra\Response\Exception cannot extend from interface Cassandra\Exception

vinothbabu13 commented:
Fix for the following error

FatalErrorException in Exception.php line 4:
Class Cassandra\Response\Exception cannot extend from interface Cassandra\Exception

vinothbabu13 commented:
@pincher2012 - Sorry i have reverted that implements to extends as it is the class and i have done one more fix now for the following error,

```
FatalErrorException in Exception.php line 4:
Class Cassandra\Response\Exception cannot extend from interface Cassandra\Exception
```

File - src\Response\Error.php
Line - 126

```
return new Exception($data['message'], $data['code']);
To
return new \Exception($data['message'], $data['code']);
```

vinothbabu13 committed:
fix for empty unpack
